Source/Boxed.DotnetNewTest/DotnetNew.cs

```diff
@@ -58,24 +59,15 @@ public static Task InstallAsync(Assembly assembly, string fileName)
         /// <param name="timeout">The timeout. Defaults to one minute.</param>
         /// <param name="showShellWindow">if set to <c>true</c> show the shell window instead of logging to output.</param>
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        /// <exception cref="ArgumentException">The provided source was null or empty.</exception>
         public static async Task InstallAsync(string source, TimeSpan? timeout = null, bool showShellWindow = false)
         {
-            if (source is null)
+            if (string.IsNullOrWhiteSpace(source))
```

Comment:
I normally split this into `ArgumentNullException` and then checking source.Length == 0 for `ArgumentException`.